dbms/src/Flash/Coprocessor/StreamWriter.h

```diff
@@ -59,8 +51,6 @@ struct StreamWriter
         if (!writer->Write(resp))
             throw Exception("Failed to write resp");
     }
-    // a helper function
-    uint16_t getPartitionNum() { return 0; }
 };
 
 using StreamWriterPtr = std::shared_ptr<StreamWriter>;
```

Comment:
using TrackedMppDataPacketPtrs = std::vector<TrackedMppDataPacketPtr>;
Then we can replace all the "TrackedMppDataPacketPtr"

Reply:
ok, updated.